src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java

```diff
@@ -142,6 +142,7 @@ public SearchParameters buildSearchParameters(SearchSQLQuery searchQuery)
             Locale locale = new Locale(action);
             sparams.addLocale(locale);
         });
+        searchQuery.getFilterQueries().forEach(sparams::addFilterQuery);
```

Comment:
What if searchQuery.getFilterQueries is null

Reply:
The pattern in the constructor prevents this: https://github.com/Alfresco/alfresco-remote-api/blob/68c9a6602f11d17989522d43c1477f270bc8d693/src/main/java/org/alfresco/rest/api/search/model/SearchSQLQuery.java